agenda-services/src/main/java/org/exoplatform/agenda/notification/builder/AgendaTemplateBuilder.java

```diff
@@ -139,43 +141,43 @@ protected MessageInfo makeMessage(NotificationContext ctx) {
     }
   }
 
-  private void addIcsFile(NotificationInfo notification, MessageInfo messageInfo) {
+  private void addIcsFile(NotificationInfo notification, MessageInfo messageInfo,ZoneId timeZone) {
```

Comment:
```suggestion
  private void addIcsFile(NotificationInfo notification, MessageInfo messageInfo, ZoneId timeZone) {
```
please, try to format the added code (not the whole file)